gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/AuditServiceImpl.java

```
@@ -75,7 +80,8 @@ public Single<Map<Object, Object>> aggregate(String domain, AuditReportableCrite
     @Override
     public Single<Audit> findById(ReferenceType referenceType, String referenceId, String auditId) {
         try {
-            return getReporter(referenceType, referenceId).findById(referenceType, referenceId, auditId)
+            return Maybe.fromOptional(getReporter(referenceType, referenceId))
+                    .flatMap(r->r.findById(referenceType, referenceId, auditId))
```

this one is OK as we want to throw an error if the search fails